Resources/Prototypes/_NF/Catalog/Fills/Paper/forms_medical.yml

```
@@ -16,119 +16,162 @@
+          [bold]Applied Topicals:[/bold]
+          \[  ] Bruise pack     \[  ] Ointment     \[  ] Gauze     \[  ] Blood pack
+          \[  ] Sutures     \[  ] Regenerative mesh
 
-        [bold]Injected Chemicals:[/bold]
-        [bullet/] Chemical, dosage: XX u
+          [bold]Injected Chemicals:[/bold]
+          [bullet/] Chemical, dosage: XX u
+          [bullet/] Chemical, dosage: XX u
 
 
 
-        [color=#AAAAAA]This Form is void without the signature and a stamp of medical specialist[/color]
+      [color=#AAAAAA]This Form is void without the signature and stamp of a medical specialist[/color]
 
 - type: entity
-  id: PaperWrittenFrontierFormPrescription
+  id: PaperWrittenMedicalFormPrescription
   name: Medical Prescription
   suffix: MappedPaper
   parent: Paper
   components:
   - type: Paper
     content: |2
-        [color=#52B4E9]◥[bold]RX[/bold]◣ [bold]Prescription[/bold][/color]
-        [color=#52B4E9]‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾[/color]
-        [head=2]Patient Information[/head]
+      [color=#52B4E9]◥[bold]N[/bold]◣ [bold]Medical Contractor Standards[/bold][/color]
+      [color=#52B4E9]‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾[/color]
+      [head=2]Medical Prescription[/head]
 
-        [bold]Name:[/bold] 
-        [bold]Species:[/bold] 
-        [bold]Gender:[/bold] 
-        [bold]Position Title:[/bold] 
-        [bold]Shuttle:[/bold] 
+      [head=3]Patient Information[/head]
+          [bold]Name:[/bold] 
+          [bold]Species:[/bold] 
+          [bold]Gender:[/bold] 
+          [bold]Position Title:[/bold] 
+          [bold]Vessel IFF:[/bold] 
 
-        [head=2]Prescription[/head]
-        [bullet/] Chemical, dosage: XX u
+      [head=3]Prescription[/head]
+          [bullet/] CHEMICAL, dosage: XX u
+          [bullet/] CHEMICAL, dosage: XX u
 
+          [bold]WRITE ANY INSTRUCTIONS HERE.[/bold]
 
 
-        [color=#AAAAAA]This Form is void without the signature and a stamp of medical specialist[/color]
+
+      [color=#AAAAAA]This Form is void without the signature and stamp of a medical specialist[/color]
 
 - type: entity # Based on form made by Radiantflash (Lykaon Rahmer)
-  id: PaperWrittenFrontierFormHealthInsurance
+  id: PaperWrittenMedicalFormHealthInsurance
   name: Health Insurance Contract
   suffix: MappedPaper
   parent: Paper
   components:
   - type: Paper
     content: |2
-        [color=#52B4E9]◥[bold]MD[/bold]◣ [bold]Health Insurance Contract[/bold][/color]
-        [color=#52B4E9]‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾[/color]
-        [head=2]Section A: General Information[/head]
+      [color=#52B4E9]◥[bold]N[/bold]◣ [bold]Medical Contractor Standards[/bold][/color]
+      [color=#52B4E9]‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾[/color]
+      [head=2]Health Insurance Contract[/head]
+
+      [head=3]Policy Holder Information[/head]
+          [bold]Name:[/bold] 
+          [bold]Species:[/bold] 
+          [bold]Gender:[/bold] 
+          [bold]Occupation:[/bold] 
```

```suggestion
          [bold]Occupation:[/bold] 
          [bold]Vessel IFF:[/bold] 
```

Forgot about any associated ship you're interested in here, potentially useful in triage.